mongoengine/queryset/transform.py

```diff
@@ -2,6 +2,7 @@
 
 import pymongo
 from bson import SON
+from mongoengine.base.fields import UPDATE_OPERATORS
 
```

Comment:
sorry for being picky, but maybe the empty line should be above and not below this line...